src/components/subscribe/subscribe.twig

```diff
@@ -0,0 +1,76 @@
+{% set _heading_tag = heading_tag|default('h2') %}
+
+<div class="{{['c-subscribe', 'js-subscribe', class|default('')]|join(' ')}}">
+  <div class="c-subscribe__controls">
+    <{{_heading_tag}}>
+      {{never_miss_article_heading|default('Never miss an article!')}}
+    </{{_heading_tag}}>
+
+    <div class="c-subscribe__controls-wrapper">
+      {% include '@cloudfour/components/button-swap/button-swap.twig' with {
+        button_class: [
+          'c-subscribe__control',
+          'js-subscribe__control',
+          notifications_btn_class|default('')
+        ]|join(' '),
+        initial_label: notifications_btn_initial_label|default('Notifications have been turned off.'),
+        initial_visual_label: notifications_btn_initial_visual_label|default('Get notifications'),
+        swapped_label: notifications_btn_swapped_label|default('Notifications have been turned on.'),
+        swapped_visual_label: notifications_btn_swapped_visual_label|default('Turn off notifications'),
+      } only %}
+
+      {% include '@cloudfour/components/button/button.twig' with {
+        href: "#subscribe-#{form_id}",
+        class: [
+          'c-subscribe__control',
+          'js-subscribe__control',
+          'js-subscribe__get-weekly-digests-btn',
+          weekly_digests_btn_class|default('')
+        ]|join(' '),
+        label: weekly_digests_btn_label|default('Get Weekly Digests'),
+        content_start_icon: weekly_digests_btn_icon|default('envelope')
+      } only %}
+    </div>
+  </div>
+
+  <form
+    id="subscribe-{{form_id}}"
+    action="{{form_action|default('https://cloudfour.us13.list-manage.com/subscribe/post?u=ce064f42c86a5982dd218d4de&amp;id=7e505a6a67')}}"
```

Comment:
I'm not sure I agree with embedding this in the front-end pattern since it's so specific to a single project. I think it's reasonable to expect this to be provided by the host project instead.

Reply:
Done! https://github.com/cloudfour/cloudfour.com-patterns/pull/1679/commits/c9e2af36d0f38ad0c46ae14e6629a96416c2ccba